src/main/java/SYSC4806/Application.java

```diff
@@ -1,12 +1,92 @@
 package SYSC4806;
 
+import SYSC4806.Model.Category;
+import SYSC4806.Model.Course;
+import SYSC4806.Model.LearningOutcome;
+import SYSC4806.Model.Program;
+import SYSC4806.Repository.CategoryRepository;
+import SYSC4806.Repository.CourseRepository;
+import SYSC4806.Repository.LearningOutcomeRepository;
+import SYSC4806.Repository.ProgramRepository;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import java.util.ArrayList;
+import java.util.List;
 
 @SpringBootApplication
 public class Application {
 
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
+    @Bean
+    public CommandLineRunner demo(CategoryRepository categoryRepository, CourseRepository courseRepository, LearningOutcomeRepository learningOutcomeRepository, ProgramRepository programRepository) {
+        return (args) -> {
+
+            // Testing purposes to connect DB to Front
+            // Can delete this setup once DB is up
+            // Hard Input of each object.
+            List<LearningOutcome> learningOutcomes = new ArrayList<>();
+            List<Course> courses = new ArrayList<>();
+            List<Program> programs = new ArrayList<>();
+
+            Category category = new Category();
+            Category category2 = new Category();
+            Category category3 = new Category();
+            LearningOutcome learningOutcome = new LearningOutcome();
+            Course course = new Course();
+            Course course2 = new Course();
+            Course course3 = new Course();
+            Program program = new Program();
+            Program program2 = new Program();
+            Program program3 = new Program();
+
+
+            learningOutcomes.add(learningOutcome);
+            courses.add(course);
+            programs.add(program);
+
+            category.setName("Programming");
+            category.setLearningOutcomes(learningOutcomes);
+
+            learningOutcome.setName("Web Application Coding");
+            learningOutcome.setCategory(category);
+            learningOutcome.setCourses(courses);
+
+            course.setName("Software Engineering Lab");
+            course.setYear(4);
+            course.setCode("4806");
+            course.setLearningOutcomes(learningOutcomes);
+            course.setPrograms(programs);
+
+            // SMELLLYYYYY
```

Comment:
Why is this Smelly? Just cause these values are hard coded?

Reply:
All of these models had initialization constructors :P You didn't have to set them all manually.

Reply:
oh shit how else do i call them when they need the List of other classes and when theyre not made yet? 

For ex/ It was lil confusing passing in a list of courses into learningoutcome constructor when courses arent made into a list yet. am i forgetting how to code, sorry been tired LOL.

Reply:
It would just cut down the code base instead of having all these lengthy setters. It would just look something like:

```java
List<LearningOutcome> learningOutcomes = new ArrayList<>();
List<Course> courses = new ArrayList<>();
List<Program> programs = new ArrayList<>();

Category category = new Category("Programming", learningOutcomes);
// I'm not sure if this should be null? You didn't set it using *.setLearningOutcomes() so I just left it null.
// This goes back to needing to define some validation parameters
Category category2 = new Category("Software Architecture", null);
```

and so on.

Reply:
^^ this works yup. Also if you wanted more references just look at the test I have for the Entity classes, they have examples for this as well